bitwarden_license/src/Portal/Models/PolicyModel.cs

```diff
@@ -56,6 +56,11 @@ public PolicyModel(PolicyType policyType, bool enabled)
                     NameKey = "SendOptions";
                     DescriptionKey = "SendOptionsDescription";
                     break;
+                
```

Comment:
Can we go ahead and remove the whitespace between these case statements? We don't format them like this anywhere else as far as I know.

Reply:
I personally like the extra space, but not enough to try and change it everywhere. Will update!

Reply:
Updated.